index.html

```diff
@@ -1,16 +1,108 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css">
 
+    <title>Anu's Cake Co.</title>
+</head>
 <body>
-    <!-- Add your markup here -->
-</body>
+    <header>
```

Comment:
Adding the logo and logo text inside the header would be good. Create a separate navbar section for nav.